code/modules/custom_ka/frames.dm

```diff
@@ -110,6 +110,26 @@
 	require_wield = TRUE
 	slot_flags = 0
 
+/obj/item/gun/custom_ka/frameB/handle_post_fire(mob/user)
+	..()
+	if(ishuman(user))
+		var/mob/living/carbon/human/H = user
+		if(H.mob_size < 10) // smaller than an unathi
```

Comment:
It might be worth to check for a (industrial) rig here aswell.

Reply:
Industrial rigs are just too common I'm afraid that these species based restrictions wouldn't be felt.
By the time you would get your hands on one of these big accelerators you would also already have an industrial rig.

Reply:
If I may comment, the amount of people who actually use the industrial rig in mining compared to the EVA variant is enough to warrant encouragement to use it.